src/test/java/org/snomed/aag/rest/AcceptanceControllerTest.java

```diff
@@ -617,92 +617,52 @@ void viewCriteriaItems_ShouldReturnItemMarkedComplete_WhenGivenBranchHasAccepted
-        givenCriteriaItemExists(taskCriteriaItemId4, true, 1, taskCriteriaItemId4, "complicated");
-        givenAcceptanceCriteriaExists(projectBranch, 1, Collections.singleton(projectCriteriaItemId), Set.of(taskCriteriaItemId1, taskCriteriaItemId2));
-        givenCriteriaItemSignOffExists(taskBranch, projectCriteriaItemId);
-        givenBranchDoesExist(System.currentTimeMillis(), buildMetadataWithAuthorFlag("complex", false));
-
-        //when
-        ResultActions resultActions = mockMvc.perform(get(requestUrl));
-        Set<CriteriaItem> criteriaItems = toProjectAcceptCriteria(getResponseBody(resultActions)).getCriteriaItems();
-
-        //then
-        assertEquals(4, criteriaItems.size()); // Everything returned as no enabled flags to remove
+    void viewCriteriaItems_ShouldReturnPACWithExpectedCriteria_WhenGivenVariousConfigurations() throws Exception {
+        /*
+         * Scenario             Project has Criteria            Criteria is 'mandatory'         Criteria has 'enabledByFlag'   Branch Flags    Enabled
+         * A                    Y                               -                               -                               -               Y
+         * B                    -                               Y                               -                               -               Y
+         * C                    Y                               -                               complex                         complex=true    Y
+         * D                    -                               Y                               complex                         complex=true    Y
+         * E                    -                               -                               complex                         complex=true    N
+         * F                    -                               Y                               complex                         -               N
+         * G                    Y                               -                               thing                           complex=true    N
+         * H                    -                               Y                               complex                         complex=false   N
+         * */
+        assertViewingPACRequirement("A", true, false, null, null, true);
+        assertViewingPACRequirement("B", false, true, null, null, true);
+        assertViewingPACRequirement("C", true, false, "complex", Pair.of("complex", true), true);
+        assertViewingPACRequirement("D", false, true, "complex", Pair.of("complex", true), true);
+        assertViewingPACRequirement("E", false, false, "complex", Pair.of("complex", true), false);
+        assertViewingPACRequirement("F", false, true, "complex", null, false);
+        assertViewingPACRequirement("G", true, false, "thing", Pair.of("complex", true), false);
+        assertViewingPACRequirement("H", false, true, "complex", Pair.of("complex", false), false);
```

Comment:
A number of tests have been removed as they were testing older logic. The tests have been replaced with this one, which is the test equivalent of the table of viewing requirements mentioned in the ticket.